src/azure-cli/azure/cli/command_modules/acs/_client_factory.py

```diff
@@ -58,9 +58,7 @@ def get_auth_management_client(cli_ctx, scope=None, **_):
 
 
 def get_container_service_client(cli_ctx, **_):
-    from azure.mgmt.containerservice import ContainerServiceClient
-
-    return get_mgmt_service_client(cli_ctx, ContainerServiceClient)
+    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_CONTAINERSERVICE)
```

Comment:
Please avoid positional argument here for resource type info.

Reply:
@Juliehzl Could you please provide further guidelines here? I followed the same pattern as other services in the files like:

def cf_resources(cli_ctx, subscription_id=None):
    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES,
                                   subscription_id=subscription_id).resources


def cf_container_registry_service(cli_ctx, subscription_id=None):
    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_CONTAINERREGISTRY,
                                   subscription_id=subscription_id)